src/lib.rs

```diff
@@ -1465,6 +1472,40 @@ impl RelativePath {
 
     /// Constructs a relative path from the current path, to `path`.
     ///
+    /// This function will return the empty relative path `""` if the component
```

Comment:
This function will return the empty `RelativePath`, `""` if the base (or origin) path contains unnamed components like `..` that would have to be traversed to reach the destination.

Reply:
👍